packages/wrangler/src/__tests__/parse.test.ts

```diff
@@ -443,3 +444,37 @@ describe("searchLocation", () => {
 		});
 	});
 });
+
+describe("parseByteSize", () => {
+	it("should calculate valid byte sizes", () => {
+		const cases: [string, number][] = [
+			["3", 3],
+			["3B", 3],
+			["1.3Kb", 1300],
+			["1.3kib", 1331],
+			["42MB", 42_000_000],
+			["42mib", 44_040_192],
+			["0.8MB", 800_000],
+			["0.8MiB", 838_860],
+			["2 GB", 2_000_000_000],
+
+			// If the b/ib suffix is omitted, assume non-binary units
+			["2G", 2_000_000_000],
+			["2 giB", 2_147_483_648],
```

Comment:
this entry does end with `iB` right? so, is it in the correct section of cases? (or should it be above the `b/ib` comment?)

Reply:
Yes thank you! Must have added or modified this test at some point after making the comment. Moved as suggested.